seed/python-sdk/undiscriminated-unions/snippet-templates.json

```diff
@@ -46,7 +46,7 @@
                                         "templateString": "$FERN_INPUT",
                                         "templateInputs": [
                                             {
-                                                "location": "BODY",
+                                                "location": "RELATIVE",
```

Comment:
Whats the difference between body and relative ?

Reply:
This should be documented in FDR but basically it's for convenience to declare something as relative with a passed in payload, vs giving the full path within the object

It's mostly used for iterable types to say this is the path within this chunk

But for unions it's nice because you can debug the specific payload easier